### ISSUE TITLE
Invalid aria role on back to top button

### ISSUE THREAD

**Issue by benoitlord** [benoitlord/portfolio_nuxt#8]
### Issue Description
Vérifier que l’attribut role a une valeur valide pour cet élément

### Element location
`#bouton-haut`

### Element source
`<a data-v-1c0ebc08="" id="bouton-haut" href="#" role="complementary" class="visible">`

### To solve this violation, you need to...
Fix the following:
- Le rôle ARIA complementary n’est pas autorisé pour l’élément donné
